setup.py

```diff
@@ -10,6 +12,16 @@
 if sys.version_info[0] < 3:
   sys.exit('ERROR: Python < 3 is unsupported.')
 
+platform.system()
```

Comment:
Running this install in the setup script is pretty aggressive. In fact, this code is only reachable if python3 is already installed, so it's basically unreachable. Maybe you could update the error above to say `Run brew install python3 to install on Mac OS`.

Also FWIW, they may have python3 installed but just be invoking this script through python2. In that case, the `brew install` is also just noise.